pkg/apis/machine/validation/awsmachineclass.go

```diff
@@ -125,6 +125,9 @@ func validateBlockDevices(blockDevices []machine.AWSBlockDeviceMappingSpec, fldP
 		if blockDevices[0].Ebs.VolumeType == "" {
 			allErrs = append(allErrs, field.Required(fldPath.Child("ebs.volumeType"), "Please mention a valid ebs volume type"))
 		}
+		if blockDevices[0].Ebs.VolumeType == "io1" && blockDevices[0].Ebs.Iops == 0 {
```

Comment:
Can we change this to an else if block? Also could we change the validation iops and size to <= 0 instead of == 0